### PR TITLE
g++, gcc, java, json5, python: add Nepali translation

### PR DESCRIPTION
**Pull request by animeshdhakal**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Welcome to tldr, Thanks for your contribution. 

Translate paths and other command arguments inside `{{}}`.

**Comment by kbdharun** [comment]
If possible can you add Nepali words to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) in a new PR above `nl` as it will be a helpful reference for reviewers and other translators in future?

> Thanks in advance

**Comment by animeshdhakal** [comment]
> Welcome to tldr, Thanks for your contribution.
> 
> Translate paths and other command arguments inside `{{}}`.

Updated the Translations

File: pages.ne/common/g++.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# g++
+
+> C++ स्रोत फाइलहरू कम्पाइल गर्छ।
+> GCC को भाग (GNU कम्पाइलर संग्रह).
+> थप जानकारी: <https://gcc.gnu.org>.
+
+- स्रोत कोड फाइललाई कार्यान्वयनयोग्य बाइनरीमा कम्पाइल गर्नुहोस्:
+
+`g++ {{स्रोतफाइल/को/पथ}} -o {{आउटपुट/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/g++.md:9]:
```suggestion
`g++ {{स्रोतफाइल.cpp/को/पथ}} -o {{आउटपुट/को/पथ}}`
```

If possible can you mention the executable too in the output path?
Update this throughout all pages in this PR.

File: pages.ne/common/g++.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- सामान्य चेतावनीहरू प्रदर्शन गर्नुहोस्:
+
+`g++ {{स्रोतफाइल/को/पथ}} -Wall -o {{आउटपुट/को/पथ}}`
+
+- (C++98/C++11/C++14/C++17) को लागि कम्पाइल गर्न भाषा मानक छान्नुहोस्:
+
+`g++ {{स्रोतफाइल/को/पथ}} -std={{c++98|c++11|c++14|c++17}} -o {{आउटपुट/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/g++.md:17]:
```suggestion
`g++ {{स्रोतफाइल.cpp/को/पथ}} -std={{c++98|c++11|c++14|c++17}} -o {{आउटपुट/को/पथ}}`
```

File: pages.ne/common/g++.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- स्रोत फाइल भन्दा फरक मार्गमा अवस्थित पुस्तकालयहरू समावेश गर्नुहोस्:
+
+`g++ {{स्रोतफाइल/को/पथ}} -o {{आउटपुट/को/पथ}} -I{{हेडर/को/पथ}} -L{{लाइब्ररी/को/पथ}} -l{{लाइब्ररी को नाम}}`
```
Comment (kbdharun) [inline pages.ne/common/g++.md:21]:
```suggestion
`g++ {{स्रोतफाइल/को/पथ.cpp}} -o {{आउटपुट/को/पथ}} -I{{हेडर/को/पथ}} -L{{लाइब्ररी/को/पथ}} -l{{लाइब्ररी_को_नाम}}`
```

File: pages.ne/common/g++.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- एक्जिक्युटेबल बाइनरीमा बहु स्रोत कोड फाइलहरू कम्पाइल र लिङ्क गर्नुहोस्:
+
+`g++ -c {{स्रोतफाइल_१/को/पथ स्रोतफाइल_२/को/पथ ...}} && g++ -o {{आउटपुट/को/पथ}} {{आउटपुट_१/को/पथ आउटपुट_२/को/पथ ...}}`
```
Comment (kbdharun) [inline pages.ne/common/g++.md:25]:
```suggestion
`g++ -c {{स्रोतफाइल_१.cpp/को/पथ स्रोतफाइल_२.cpp/को/पथ ...}} && g++ -o {{आउटपुट/को/पथ}} {{आउटपुट_१/को/पथ आउटपुट_२/को/पथ ...}}`
```

File: pages.ne/common/g++.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+> GCC को भाग (GNU कम्पाइलर संग्रह).
+> थप जानकारी: <https://gcc.gnu.org>.
+
+- स्रोत कोड फाइललाई कार्यान्वयनयोग्य बाइनरीमा कम्पाइल गर्नुहोस्:
+
+`g++ {{स्रोतफाइल/को/पथ}} -o {{आउटपुट/को/पथ}}`
+
+- सामान्य चेतावनीहरू प्रदर्शन गर्नुहोस्:
+
+`g++ {{स्रोतफाइल/को/पथ}} -Wall -o {{आउटपुट/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/g++.md:13]:
```suggestion
`g++ {{स्रोतफाइल.cpp/को/पथ}} -Wall -o {{आउटपुट/को/पथ}}`
```

File: pages.ne/common/gcc.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`gcc {{स्रोतफाइल_१/को/पथ स्रोतफाइल/को/पथ_२ ...}} -o {{आउटपुट/को/पथ}}`
+
+- आउटपुटमा सामान्य चेतावनीहरू, डिबग प्रतीकहरू देखाउनुहोस्:
+
+`gcc {{स्रोतफाइल/को/पथ}} -Wall -Og -o {{आउटपुट/को/पथ}}`
+
+- फरक मार्गबाट ​​पुस्तकालयहरू समावेश गर्नुहोस्:
+
+`gcc {{स्रोतफाइल/को/पथ}} -o {{आउटपुट/को/पथ}} -I{{हेडर/को/पथ}} -L{{लाइब्ररी/को/पथ}} -l{{लाइब्ररी को नाम}}`
```
Comment (kbdharun) [inline pages.ne/common/gcc.md:16]:
```suggestion
`gcc {{स्रोतफाइल.c/को/पथ}} -o {{आउटपुट/को/पथ}} -I{{हेडर/को/पथ}} -L{{लाइब्ररी/को/पथ}} -l{{लाइब्ररी को नाम}}`
```

File: pages.ne/common/gcc.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- एसेम्बलर निर्देशनहरूमा स्रोत कोड कम्पाइल गर्नुहोस्:
+
+`gcc -S {{स्रोतफाइल/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/gcc.md:20]:
```suggestion
`gcc -S {{स्रोतफाइल.c/को/पथ}}`
```

File: pages.ne/common/gcc.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- लिङ्क नगरी वस्तु फाइलमा स्रोत कोड कम्पाइल गर्नुहोस्
+
+`gcc -c {{स्रोतफाइल/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/gcc.md:24]:
```suggestion
`gcc -c {{स्रोतफाइल.c/को/पथ}}`
```

File: pages.ne/common/json5.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`json5 {{निवेशफाइल/को/पथ}} --out-file {{निर्गमफाइल/को/पथ}}`
+
+- JSON5 फाइल प्रमाणित गर्नुहोस्:
+
+`json5 {{निवेशफाइल/को/पथ}} --validate`
+
+- इन्डेन्ट द्वारा (वा ट्याबहरूको लागि "t") गर्न खाली ठाउँहरूको संख्या निर्दिष्ट गर्नुहोस्:
+
+`json5 --space {{इन्डेन्ट मात्रा}}`
```
Comment (kbdharun) [inline pages.ne/common/json5.md:24]:
```suggestion
`json5 --space {{इन्डेन्ट_मात्रा}}`
```

File: pages.ne/common/json5.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+`json5 {{निवेशफाइल/को/पथ}}`
+
+- JSON5 फाइललाई निर्दिष्ट JSON फाइलमा रूपान्तरण गर्नुहोस्:
+
+`json5 {{निवेशफाइल/को/पथ}} --out-file {{निर्गमफाइल/को/पथ}}`
+
+- JSON5 फाइल प्रमाणित गर्नुहोस्:
+
+`json5 {{निवेशफाइल/को/पथ}} --validate`
```
Comment (kbdharun) [inline pages.ne/common/json5.md:20]:
```suggestion
`json5 {{निवेशफाइल.json5/को/पथ}} --validate`
```

File: pages.ne/common/json5.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+`echo {{निवेश}} | json5`
+
+- JSON5 फाइललाई JSON र आउटपुटलाई stdout मा रूपान्तरण गर्नुहोस्:
+
+`json5 {{निवेशफाइल/को/पथ}}`
+
+- JSON5 फाइललाई निर्दिष्ट JSON फाइलमा रूपान्तरण गर्नुहोस्:
+
+`json5 {{निवेशफाइल/को/पथ}} --out-file {{निर्गमफाइल/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/json5.md:16]:
```suggestion
`json5 {{निवेशफाइल.json5/को/पथ}} --out-file {{निर्गमफाइल/को/पथ}}`
```

File: pages.ne/common/json5.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+> JSON5 फाइलहरूलाई JSON मा रूपान्तरण गर्नको लागि कमाण्ड-लाइन उपकरण।
+> थप जानकारी: <https://json5.org>.
+
+- JSON5 stdin लाई JSON stdout मा रूपान्तरण गर्नुहोस्:
+
+`echo {{निवेश}} | json5`
+
+- JSON5 फाइललाई JSON र आउटपुटलाई stdout मा रूपान्तरण गर्नुहोस्:
+
+`json5 {{निवेशफाइल/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/json5.md:12]:
```suggestion
`json5 {{निवेशफाइल.json5/को/पथ}}`
```

File: pages.ne/common/python.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+> पाइथन भाषा अनुवादक.
+> थप जानकारी: <https://www.python.org>.
+
+- REPL सुरु गर्नुहोस् (अन्तर्क्रियात्मक शेल):
+
+`python`
+
+- एक विशिष्ट पाइथन फाइल कार्यान्वयन गर्नुहोस्:
+
+`python {{फाइल/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/python.md:12]:
```suggestion
`python {{फाइल.py/को/पथ}}`
```

File: pages.ne/common/python.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+- एक विशिष्ट पाइथन फाइल कार्यान्वयन गर्नुहोस् र REPL सुरु गर्नुहोस्:
+
+`python -i {{फाइल/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/python.md:16]:
```suggestion
`python -i {{फाइल.py/को/पथ}}`
```

File: pages.ne/common/python.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+`python -m {{मोडयुल}} {{तर्कहरू}}`
+
+- `pip` प्रयोग गरेर प्याकेज स्थापना गर्नुहोस्:
+
+`python -m {{pip}} install {{प्याकेजको नाम}}`
+
+- अन्तरक्रियात्मक रूपमा पाइथन स्क्रिप्ट डिबग गर्नुहोस्:
+
+`python -m {{pdb}} {{फाइल/को/पथ}}`
```
Comment (kbdharun) [inline pages.ne/common/python.md:32]:
```suggestion
`python -m {{pdb}} {{फाइल.py/को/पथ}}`
```

**Comment by animeshdhakal** [comment]
Added the Extension to Translation

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.ne/common/json5.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+
+`json5 {{निवेशफाइल.json5/को/पथ}} --out-file {{निर्गमफाइल.json/को/पथ}}`
+
+- JSON5 फाइल प्रमाणित गर्नुहोस्:
+
+`json5 {{निवेशफाइल.json5/को/पथ}} --validate`
+
+- इन्डेन्ट द्वारा (वा ट्याबहरूको लागि "t") गर्न खाली ठाउँहरूको संख्या निर्दिष्ट गर्नुहोस्:
+
+`json5 --space {{इन्डेन्ट मात्रा}}`
```
Comment (navarroaxel) [inline pages.ne/common/json5.md:24]: We use `snake_case` for placeholder:
```suggestion
`json5 --space {{इन्डेन्ट_मात्रा}}`
```

**Pull request closed without merging**